docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1327,6 +1327,311 @@ The ACS will be used for V&V testing. It is defined as:
+
+    Also need to address complex family structures 
+
+
+Not everyone who receives a W2 or 1099 will end up filing taxes. 
+However, those who do not are concentrated in low incomes for whom 
+taxes are not required. Currently, we will chose to have all those 
+who are legally required to file taxes, file taxes. This is a 
+limitation and is listed below. 
+
+For simulants that receive below the minimum income, 42.14% will 
+still file taxes. [Cilke_1998]_ The remainder will not. The minimum 
+income is based on the household structure and is listed in the table below. 
+We will not model persistence year to year. 
+
+**In the current model, no one will be low income, this will be changed later.** 
+
+.. list-table:: Minimum Income  
+  :widths: 20 20 
+  :header-rows: 0
+
+  * - Simulant Type 
+    - Minimum Income 
+  * - Single filing, under 65 
+    - $12,550 
+  * - Single filing, over 65 
+    - $14,250
+  * - Married joint filing, both under 65 
+    - $25,100
+  * - Married joint filing, one under 65 
+    - $26,450
+  * - Married joint filing, both over 65 
+    - $27,800
+  * - Married separate filing 
+    - $5
+
+
+Based on the household structure, the following rules can be applied 
+for who files taxes: 
+
+- Assume that 95% of spouses file jointly, this can be randomly assigned. [Nolo]_ Others will file separately. 
+    * The only spouses we will recognize are [Reference person, Opp-sex spouse] and [Reference person, Same-sex spouse]. 
+    * The reference person will submit the form, the spouse will be listed as the joint filer. 
+    * There does not need to be persistence in who files jointly, it can be re-drawn each year. 
+- All other non-married simulants in a household with a W2 or 1099 will file separately, based on the income rules above (e.g., a low-income earner in a house with other earners will be randomly assigned to file or not file, independent of others in the household). Please note that simulants can BOTH be claimed as a dependent AND file their own taxes. 
+- All simulants eligible to be dependents will be assigned to a relative within the household 
+    * If there someone listed as the dependent's parents and they are filing taxes, they will be assigned to their parent 
+    * If there is not a parent, they will randomly be assigned to a tax-filing relative (not housemate or other non-relative) in the household 
+- A simulant eligble to be a dependent must: 
+    * Not be a "housemate/roommate" or "other nonrelative" to whoever is claiming them 
+    * The dependent's income must be below $4300 
+- For simulants outside of the household, they will be claimed as a dependent by their parent until age 19 OR age 24 if they are in group quarters for college 
+    * For simulants born in the simulation, they should have at least one parent tracked 
+    * For simulants not born in the simulantion, information on initializing parents is being added shortly 
```

Comment:
when you do add this info, if you end up deciding to track multiple parents, will you include info on how to decide which parent claims?

Reply:
Good point! I added something about this here.